docker/node-runner.sh

```diff
@@ -1,23 +1,26 @@
 #!/bin/bash
 
 # Inits node configuration and runs the node.
+# e -> exit immediately, u -> treat unset variables as errors and immediately, o -> sets the exit code to the rightmost command 
+set -euo pipefail
 
-set -euox pipefail
-
+# within the container, $HOME=/home/cheqd
 CHEQD_ROOT_DIR="$HOME/.cheqdnode"
 
 # Init node config directory
 if [ ! -d "${CHEQD_ROOT_DIR}/config" ]
 then
     echo "Node config not found. Initializing."
-    cheqd-noded init $NODE_MONIKER --home 
+    cheqd-noded init $NODE_MONIKER
+	curl -sSL "https://raw.githubusercontent.com/cheqd/cheqd-node/main/persistent_chains/${NETWORK}/genesis.json" > ${CHEQD_ROOT_DIR}/config/genesis.json
```

Comment:
What if someone would like to run the container with custom genesis and seeds? For example, it could be useful in our docker-compose-based localnet.